ipaserver/install/kra.py

```diff
@@ -47,9 +49,9 @@ def install_check(api, replica_config, options):
 
     if replica_config is not None:
         if not api.Command.kra_is_enabled()['result']:
-            raise RuntimeError(
-                "KRA is not installed on the master system. Please use "
-                "'ipa-kra-install' command to install the first instance.")
+            root_logger.info("KRA is not installed on the master system. "
```

Comment:
Please remove this logging. Note that `root_logger` should generally not be used (although it is unfortunately used very much throughout IPA a lot).
I also believe that a user does not care about this message.